### PR TITLE
Renamed cml to theta

### PR DESCRIPTION
**Pull request by anisham197**
Tested the change by adding a small model to a repository and pushing it to remote.

### CODE REVIEW

**Comment by muqeeth** [comment]
How did the git knows the file name has changed when you renamed? I thought it would tell git_cml files are deleted and git_theta files are added.

**Comment by blester125** [comment]
> git knows the file name has changed when you renamed?

If you do `git mv` then git tracks it as a rename instead of a delete and add. I think sometimes it will also automatically do it sometimes but `git mv` is the most reliable way.

File: examples/git_cml_example.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ git init
```
Comment (nkandpa2) [inline examples/git_cml_example.md:9]: Could we rename this file as well to `git_theta_example.md`.

**Review by nkandpa2 — APPROVED** [review]

**Comment by muqeeth** [comment]
Thanks Brian for clarifying

**Pull request merged**